assets/js/components/Popup.js

```diff
@@ -222,13 +222,31 @@ module.exports = class Popup {
     return utils.wrapElements(children, 'div', 'popup');
   }
 
+  hasAuthorClassName() {
+    const authorLink = document.querySelector('.author_link');
+    if (!authorLink) {
+      return false;
+    }
+
+    const authorLinkParent = authorLink.parentElement.parentElement;
+    const hasAuthorItem = authorLinkParent.classList.contains('author_list_item');
+    return hasAuthorItem;
```

Comment:
Why are we simply not returning true if `.author_link` is found in the document?

Reply:
It feels a bit wrong to be making this change because Popup.js is supposed to be a generic behaviour that can be applied to any component and not need to have override's for specific cases.

We should add examples in for the now 3 article page templates:

- new with content aside
- feature article
- insight or editorial

We should try and keep all of the override code together and we should make it clear that it is because of the position absolute.